clients/java/client/src/main/java/org/camunda/bpm/client/exception/NotAcquiredException.java

```diff
@@ -17,16 +17,14 @@
 package org.camunda.bpm.client.exception;
 
 /**
- * <p>Exception is thrown if if the task's most recent lock could not be acquired</p>
- *
- * @author Tassilo Weidner
+ * @deprecated since this exception has been replaced by {@link BadRequestException} and will be removed in version 7.19.
```

Comment:
Just a comment, not hinting at changes here: It would have been nice if we could use the Java 9 optional arguments for the `@Deprecated` annotation here. (https://www.baeldung.com/java-deprecated#optional-attributes)

Reply:
Wasn't aware of it. Good hint. I'll look into it 👍